pkg/rtc/participant.go

```diff
@@ -1152,6 +1152,7 @@ func (p *ParticipantImpl) onMediaTrack(track *webrtc.TrackRemote, rtpReceiver *w
 	p.params.Logger.Infow("mediaTrack published",
 		"kind", track.Kind().String(),
 		"trackID", publishedTrack.ID(),
+		"webrtcTrackID", track.ID(),
```

Comment:
is this equivalent to our cid?

Reply:
It is sdp cid, equal to signal cid for chrome/safari, but not equal for firefox